src/pages/viewer-app/viewer-app.tsx

```diff
@@ -564,7 +564,27 @@ export const ViewerApp = () => {
           preventTransitions={preventTransitions}
         />
       )}
-      {selectedBaseMapId === "ArcGis" && <ArcgisWrapper />}
+      {selectedBaseMapId === "ArcGis" && (
+        <ArcgisWrapper
```

Comment:
It seems `DeckGlWrapper` and `ArcgisWrapper` have same input. You can

```
const MapWrapper = selectedBaseMapId === "ArcGis" ? ArcgisWrapper : DeckGlWrapper;
...
return ......
  <MapWrapper ...
```

Reply:
1. Aligned input parameters between DeckGlWrapper and ArcgisWrapper.
2.  Added recommended construction:
const MapWrapper = selectedBaseMapId === "ArcGis" ? ArcgisWrapper : DeckGlWrapper;
...
return ......
  <MapWrapper ...